core/shared/src/main/scala/scalaz/zio/Managed.scala

```diff
@@ -1,6 +1,12 @@
 package scalaz.zio
 
 sealed abstract class Managed[+E, +R] extends Serializable { self =>
+  type R0 <: R
+
+  def acquire: IO[E, R0] = IO.never
```

Comment:
Why not keep this and `release` abstract?

Reply:
@iravid thanks for reviewing, will take a look!

Reply:
@iravid please re-review.